app/(client)/event-calendar/page.tsx

```diff
@@ -0,0 +1,208 @@
+            >
+              {isDesc ? (
+                <SortAsc className="scale-150" />
+              ) : (
+                <SortDesc className="scale-150" />
+              )}
+            </Button>
+          </div>
+        </div>
+        <div className="w-full flex items-center justify-center max-[1280px]:px-3">
+          <div className="grid grid-cols-5 gap-5 gap-y-10 w-full max-[1280px]:flex max-[1280px]:flex-col max-[1280px]:gap-y-3">
+            <Event
+              title={'День перваша'}
+              description={
+                'Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис '
+              }
+              location={'КПІ'}
+              tag={'перваш'}
+              date={'31.09.2024'}
+              time={'15:00-16:00'}
+            />
+            <Event
+              title={'День перваша'}
+              description={
+                'Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис '
+              }
+              location={'КПІ'}
+              tag={'перваш'}
+              date={'31.09.2024'}
+              time={'15:00-16:00'}
+            />
+            <Event
+              title={'День перваша'}
+              description={
+                'Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис '
+              }
+              location={'КПІ'}
+              tag={'перваш'}
+              date={'31.09.2024'}
+              time={'15:00-16:00'}
+            />
+            <Event
+              title={'День перваша'}
+              description={
+                'Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис '
+              }
+              location={'КПІ'}
+              tag={'перваш'}
+              date={'31.09.2024'}
+              time={'15:00-16:00'}
+            />
+            <Event
+              title={'День перваша'}
+              description={
+                'Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис '
+              }
+              location={'КПІ'}
+              tag={'перваш'}
+              date={'31.09.2024'}
+              time={'15:00-16:00'}
+            />
+            <Event
+              title={'День перваша'}
+              description={
+                'Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис Короткий опис '
+              }
+              location={'КПІ'}
+              tag={'перваш'}
+              date={'31.09.2024'}
+              time={'15:00-16:00'}
```

Comment:
all these elements <Event/> are the same, only the filling is different. All data must be taken from the backend, and there must be a map through all the data

Reply:
I am told to add real data after the pr is merged. So there is no real data yet.